stories/Pagination/Pagination.story.jsx

```diff
@@ -0,0 +1,297 @@
+        <StoryContainer>
+          <Title as="h2">Importing Pagination</Title>
+          <SimpleHighlight>{`import { Pagination } from '@catho/quantum';`}</SimpleHighlight>
+
+          <p>
+            An adaptive Pagination component, if viewport width is below small
+            breakpoint value it will renders a mobile friendly component, if
+            not, will render an dotted Pagination component.
+          </p>
+
+          <p>
+            <strong>
+              It&apos;s highly recommended to avoid pagination and use infinite
+              scroll in small devices.
+            </strong>
+          </p>
+
+          <Title as="h2">Examples</Title>
+
+          <p>
+            If you want to control the state of pagination, set{' '}
+            <code>onPageClick</code>
+            prop, if you want the <code>{'<a>'}</code> element controls the
+            pagination, set <code>pageHref</code> prop.
+          </p>
+          <p>Check the examples below:</p>
+
+          <Title as="h3">Simple usage</Title>
+          <Row>
+            <Col medium={5}>
+              <SimpleHighlight>{`<Pagination totalPages={6} />`}</SimpleHighlight>
+            </Col>
+            <Col medium={7}>
+              <Pagination
+                totalPages={6}
+                infoFormatter={(activePage, totalPages) => (
+                  <>
+                    <strong>{activePage}</strong> de{' '}
+                    <strong>{totalPages}</strong>
+                  </>
+                )}
+              />
+            </Col>
+          </Row>
+
+          <Title as="h3">With several pages and active page</Title>
+          <Row>
+            <Col medium={5}>
+              <SimpleHighlight>{`<Pagination totalPages={10} activePage={5} />`}</SimpleHighlight>
+            </Col>
+            <Col medium={7}>
+              <Pagination totalPages={10} activePage={5} />
+            </Col>
+          </Row>
+
+          <Title as="h3">Custom previous and next button texts</Title>
+          <Row>
+            <Col medium={5}>
+              <p>
+                It&apos;s possible to change the text from &quot;Previous&quot;
+                and &quot;Next&quot; buttons through <code>prevButtonText</code>{' '}
+                and <code>nextButtonText</code> props.
+              </p>
+              <SimpleHighlight>{`<Pagination
+  prevButtonText="Previous Page"
+  nextButtonText="Next Page"
+  totalPages={10}
+  activePage={5}
+/>`}</SimpleHighlight>
+            </Col>
+            <Col medium={7}>
+              <Pagination
+                prevButtonText="Previous Page"
+                nextButtonText="Next Page"
+                totalPages={10}
+                activePage={5}
+              />
+            </Col>
+          </Row>
+
+          <Title as="h3">Helper text</Title>
+          <Row>
+            <Col medium={5}>
+              <p>
+                When viewport is below <code>small</code> breakpoint, then,
```

Comment:
This info is duplicated at the top of this page. Consider leaving only one.

Reply:
Done.
Thanks!